src/Serilog.Sinks.MSSqlServer/Sinks/MSSqlServer/ColumnOptions.cs

```diff
@@ -10,7 +10,7 @@ namespace Serilog.Sinks.MSSqlServer
     /// </summary>
     public class ColumnOptions
     {
-        private ICollection<StandardColumn> _store;
+        ICollection<StandardColumn> _store;
```

Comment:
Removed the "redundant" `private` modifier, since it appears that's the style for Serilog.